lnd_test.go

```diff
@@ -3945,7 +3964,7 @@ func testRevokedCloseRetributionRemoteHodl(net *lntest.NetworkHarness,
 	// maxFundingAmount (2^24) satoshis value.
 	ctxt, _ := context.WithTimeout(ctxb, timeout)
 	chanPoint := openChannelAndAssert(
-		ctxt, t, net, dave, carol, chanAmt, pushAmt,
+		ctxt, t, net, net.Alice, carol, chanAmt, pushAmt, false,
```

Comment:
`dave` has been changed to `net.Alice` here. That's probably why Travis is failing.

Reply:
Nice catch! Not sure how this happened 🤔